src/libsystemd/sd-bus/bus-error.c

```diff
@@ -465,7 +465,7 @@ _public_ int sd_bus_error_set_errno(sd_bus_error *e, int error) {
         if (!e)
                 return -error;
         if (error == 0)
-                return -error;
+                return 0;
```

Comment:
commit message does not mention this change - was it intended for another?

Reply:
It's a trivial cleanup, more of a style change really.